packages/astro/src/@types/astro.ts

```diff
@@ -238,6 +238,22 @@ export interface AstroGlobal<
 	response: ResponseInit & {
 		readonly headers: Headers;
 	};
+	/**
+	 * get an action result on the server when using a form POST.
```

Comment:
```suggestion
	 * Get an action result on the server when using a form POST.
```
I don't usually review these, but just checking that you intended lower case here!